server/template.yml

```diff
@@ -71,7 +71,20 @@ Resources:
           Properties:
             Queue: !GetAtt PostProcessHandlerQueue.Arn
 
-    
+  FileCompress:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: fileCompress/
+      Handler: app.lambda_handler
+      Runtime: python3.9
+      Timeout: 10
+      Events:
+        FileCompress:
+          Type: Api
+          Properties:
+            Path: /fileCompress
+            Method: get
+
```

Comment:
you can give your lambda function permission here with the `Policies` property. See the PostProcessHandler function in the template.